### PR TITLE
Initial query docs

### PR DESCRIPTION
**Pull request by johan-bjareholt**

### CODE REVIEW

File: examples/client.py [hunk 1]
```diff
@@ -57,10 +57,11 @@
...
     sleep(1)
 
-# Synchronous example, insert a shutdown event
-shutdown_data = {"label": "shutdown"}
-shutdown_event = Event(timestamp=now, data=shutdown_data)
-inserted_event = client.insert_event(bucket_id, shutdown_event)
+# Synchronous example, insert an event
+event_data = {"label": "non-heartbeat event"}
+now = datetime.now(timezone.utc)
+event = Event(timestamp=now, data=shutdown_data)
```
Comment (ErikBjare) [inline examples/client.py]: `data=shutdown_data` -> `data=event_data`

Comment (johan-bjareholt) [reply examples/client.py]: Hmm weird, was pretty certain that i had tested that code but apparently not. Fixed.

File: writing-watchers.rst [hunk 2]
```diff
@@ -3,6 +3,8 @@ Writing your first watcher
 
 Writing watchers for ActivityWatch is pretty easy, all you need is the :code:`aw-client` library.
 
+**NOTE:** These examples runs the client in *testing* mode, which means that it will try to connect to a aw-server in testing mode on the port 5666 instead of the normal 5600.
```
Comment (ErikBjare) [inline writing-watchers.rst]: Use a `.. note::` block here :)

Comment (johan-bjareholt) [reply writing-watchers.rst]: Fixed

File: extending.rst [hunk 3]
```diff
@@ -9,31 +9,27 @@ We've tried to make things easy for you (and ourselves) so here's some advice on
 Collecting more data
 --------------------
 
+ActivityWatch is written to be flexible to be able to gather most types of data.
+Except for the included aw-watcher-window and aw-watcher-afk which tracks your application usage and are included by default in ActivityWatch, there are additional so-called :doc:`watchers` for activitywatch.
+
+If there is some kind of data you would want to track but are not
```
[the remark this answers is not in the document]
Comment (johan-bjareholt) [reply extending.rst]: Fixed

**Comment by johan-bjareholt** [comment]
There is still a lot more to add, but since this is a significant improvement and I don't have time to work on it right now I'll merge this for now.

**Comment by ErikBjare** [comment]
Nice!

**Pull request merged**